src/frontend/handlers.go

```diff
@@ -355,6 +362,21 @@ func (fe *frontendServer) chooseAd(ctx context.Context, ctxKeys []string, log lo
 	return ads[rand.Intn(len(ads))]
 }
 
+// convertAllCurrencies converts the currency for all products into every other currency
```

Comment:
nit: no need to repeat method name in the comment - just focus on what the code below does

Reply:
fixed!